src/gvars.c

```diff
@@ -226,6 +226,67 @@ static Obj             FopiesGVars;
 
 #endif
 
+// FlagsGVars contains information about global variables as a single INTOBJ
+// Once turned into a C int, this information is:
+//
+// Bits 0 and 1 - A value of type GVarWriteFlag which denotes if the variable
+// is ReadOnly, Mutable or Constant
```

Comment:
So I am mostly happy with this PR now, except for the use of the term "mutable", which I feel is already overloaded in GAP with other meanings. But please don't hesitate to argue otherwise :-)

Reply:
I could have sworn we use 'mutable' for gvars, but having grepped the code that is not true. changed.